scripts/export_inspector_findings.py

```diff
@@ -169,10 +171,17 @@ def column_alpha(self, col: int) -> str:
     def findings_sort(self, item: tuple[str, list[SummaryType]]) -> tuple[int, str]:
         score = 0
         weights = {'HIGH': 1, 'CRITICAL': 10}
-        for summary in item[1]:
+        vulnerability, summaries = item
+        for summary in summaries:
             count = len(summary['resources'])
             score += count * weights.get(summary['severity'], 0)
-        return score, item[0]
+        if vulnerability.startswith('CVE-'):
+            # Best secondary-sorting effort on CVE findings, vulnerability names
+            # not prefixed with 'CVE' may reflect an inaccurate secondary order.
+            id = vulnerability.split('-')[-1]
+            padded_id = '000000'[:abs(6 - len(id))] + id
+            vulnerability = vulnerability.replace(id, padded_id)
```

Comment:
```suggestion
            vulnerability = '-'.join([prefix, padded_id])
```

Reply:
This doesn't quite work, since it omits "year" aspect of the CVE name which also needs to be considered for the secondary sort.

Reply:
rpartition() splits on the first match from the right. The year isn't omitted, it's included in the first item from rpartition.

```
>>> "CVE-2024-123".rpartition('-')
('CVE-2024', '-', '123')
```

I think the variable name `prefix` was throwing you off. how about:
```
cve_year, _, id = vulnerability.rpartition('-')
```

Reply:
I see, that's what it was.
But I still prefer my approach.